src/components/statusInfo/FullLoading/index.tsx

```diff
@@ -65,7 +65,7 @@ interface Props extends HTMLAttributes<HTMLDivElement> {
   actionButton?: ActionButtonProps | undefined
   height?: string
   icon?: IconTypes
-  message?: string
+  message?: string | React.ReactNode | undefined
```

Comment:
With `?` at the end of message we could remove `undefined` from the union?

Reply:
Remove it